packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -433,6 +434,62 @@ export default class LocusInfo extends EventsScope {
     }
   }
 
+  /**
+   * Update the participantDelta property of this object based on a list of
```

Comment:
You are updating `this.deltaParticipants` in this function.

Reply:
Will address this.